pkg/apis/servicecatalog/types.go

```diff
@@ -439,3 +439,15 @@ const (
 const (
 	FinalizerServiceCatalog string = "kubernetes-incubator/service-catalog"
 )
+
+// Parameters is a union type for supporting different ways of representing parameters used for:
+// - Instance input parameters
+// - Binding output credentials
+type Parameters struct {
+	// Reference to a secret in the namespace.
```

Comment:
@pmorie do you think it should be represented as YAML or JSON? Just found that current implementation requires YAML...